packages/components/nodes/llmcache/LLMInMemoryCache/LLMInMemoryCache.ts

```diff
@@ -0,0 +1,36 @@
+import { INode, INodeParams } from '../../../src/Interface'
```

Comment:
Tested InMemoryCache, it won't work in Flowise. Reason because the class is being reinitalized everytime, hence making the cache `Map` to be reset everytime.

For now, we can get rid of this. I have a solution in place, will open a PR after this get merged

Reply:
OK.

Reply:
In this PR, we won't be able to let users select simple cache. Will open another PR to allow users to do that. We'll have to create a global cache pool in flowise for that, similar to `ChatflowPool` in `server`

Reply:
Done! Removed InMemoryCache for now, we will do a new feature to enable the local cache.